src/jit/flowgraph.cpp

```diff
@@ -4261,33 +4261,6 @@ void        Compiler::fgFindJumpTargets(const BYTE * codeAddr,
         pSm->Start(this);
     }
 
-#ifdef FEATURE_LEGACYNETCF
```

Comment:
We do not need the NetCF quirks anymore - so it safe to delete it in the JIT. 

Disabling/deleting FEATURE_LEGACYNETCF globally breaks stuff because of it got intertangled with other ifdefs. So this cleanup is probably best done in steps.